shell/platform/fuchsia/BUILD.gn

```diff
@@ -75,6 +75,8 @@ fuchsia_host_bundle("dart_binaries") {
 }
 
 group("fuchsia") {
+  testonly = true
```

Comment:
The `testonly = true` confuses me here since it deps on our runners, which aren't testonly.